tools/e2e-commons/pages/wp-admin/blocks/simple-payments.js

```diff
@@ -37,7 +37,7 @@ export default class SimplePaymentBlock extends PageActions {
 	async waitForResponse() {
 		const testUrl = /^https?:\/\/.*%2Fwp%2Fv2%2Fjp_pay_product/;
 
-		this.page.waitForResponse( resp => testUrl.test( resp.url() ) );
+		await this.page.waitForResponse( resp => testUrl.test( resp.url() ) );
```

Comment:
I assume this is a follow-up on #27951?

Reply:
yes, correct. I missed `await` in a function call